lib/unleash/metrics_reporter.rb

```diff
@@ -24,21 +23,19 @@ def generate_report
         'specVersion': Unleash::CLIENT_SPECIFICATION_VERSION,
         'appName': Unleash.configuration.app_name,
         'instanceId': Unleash.configuration.instance_id,
-        'bucket': metrics
+        'bucket': metrics || {}
       }
     end
 
     def post
       Unleash.logger.debug "post() Report"
-
-      bucket = self.generate_report
-      if bucket.nil? && (Time.now - self.last_time < LONGEST_WITHOUT_A_REPORT) # and last time is less then 10 minutes...
+      report = self.generate_report
```

Comment:
```suggestion

      report = self.generate_report
      
```

Reply:
What's the suggestion here? Adding some space? Just making sure I'm not missing something else

Reply:
Yeah, it's no big deal, just keeping the spacing around the statement like before.